jwst/refpix/reference_pixels.py

```diff
@@ -788,279 +823,300 @@ def do_subarray_corrections(self):
         #
         #  First transform to detector coordinates
         #
-        self.DMS_to_detector()
         refdq = dqflags.pixel['REFERENCE_PIXEL']
```

Comment:
need to add
self.DMS_to_detector(0,0)
to make sure the data and dq groups are shifted/rotated to DMS coordinates

Reply:
Addressed with commit stsci-hack/jwst@47dd1d9b839fff2d14806580c18a5a1ed3d72da4.

Reply:
This should now be addressed properly with 86be1ed.

Reply:
This looks good now